distribution/src/config/jvm.options

```diff
@@ -58,6 +58,9 @@
 # result in less optimal vector performance
 20-:--add-modules=jdk.incubator.vector
 
+# Enable random access flags via madvise
+-Des.madv_random_feature_flag_enabled=true
```

Comment:
I think typically when we want to enable something by default that was previously behind a feature flag, we simply remove the feature flag. In either case, I don't think we'd want this option in `jvm.options` but rather in `SystemJvmOptions`. @rjernst might provide more insight here.

Reply:
That's right, we should remove the feature flag to "enable" it permanently. If the intent is for users to be able to disable this, then it should be moved from a feature flag to a setting.

Reply:
roger! I will just remove the flag :)